aptos-move/vm-genesis/src/lib.rs

```diff
@@ -92,8 +95,10 @@ pub fn encode_genesis_change_set(
     stdlib_module_bytes: &[Vec<u8>],
     consensus_config: OnChainConsensusConfig,
     chain_id: ChainId,
-    genesis_configs: &GenesisConfigurations,
+    genesis_config: &GenesisConfigurations,
```

Comment:
We have two options here, either rename it all to `GenesisConfiguration` or just leave these as `genesis_configs` for now.
```suggestion
    genesis_config: &GenesisConfiguration,
```

Or
```suggestion
    genesis_configs: &GenesisConfigurations,
```

Reply:
Updated GenesisConfigurations -> GenesisConfiguration